view/adminhtml/web/js/import.js

```diff
@@ -42,49 +46,103 @@ define([
                 var self = this;
                 self.element.on('click', function(e) {
                     // Disable the button during the import
-                    $(this).addClass('disabled').attr('disabled', 'disabled').text('Importing orders');
+                    $(this).addClass('disabled').attr('disabled', 'disabled').text('Importing Customers');
+
+                    self.chunkSync(0, self.options.importStatus, false);
 
-                    if(self.options.totalChunks > 0){
-                        self.options.percentage = (100 / self.options.totalChunks);
-                        self.chunkSync(0);
-                    }
                 });
             },
 
             /**
-             * Post chunk id to proceed orders to Metrilo
+             * Post chunk id to proceed data to Metrilo
              *
              * @param  {integer} chunkId
              * @return {void}
              */
-            chunkSync: function(chunkId) {
+            chunkSync: function(chunkId, importStatus, retry) {
```

Comment:
What is `importStatus`?

Reply:
What is `retry`?

Reply:
Renamed the parameters to importStatus = importType (marking the current importing type)
and retry = retryStatus (true if there was an error during the specific chunk import, triggering retry of the whole importing step)